backend/Social/SocialAPI.py

```diff
@@ -77,32 +82,15 @@ def users():
         }
         return make_response(response, 400)
     except Exception as e:
+        response = {
+            "status": "failed",
+            "err": str(e)
+        }
         print(e)
-        return {"err": "An error has occured", "status": "failed"}, 500
```

Comment:
Why you changing my error handling? The try catch catches all exception so the status code should be 500. We should not expose unexpected error messages to users, just log them